oval-schemas/windows-definitions-schema.xsd

```diff
@@ -3322,6 +3332,23 @@
                                                 <xsd:documentation>The subgroup_sid entity holds a string that represents the SID of particular subgroup in the specified group. This entity can be included multiple times in a system characteristic item in order to record that a group contains a number of different subgroups. Note that the entity_check attribute associated with EntityStateStringType guides the evaluation of entities like subgroup_sid that refer to items that can occur an unbounded number of times.</xsd:documentation>
                                           </xsd:annotation>
                                     </xsd:element>
+									  <xsd:element name="group" type="oval-def:EntityStateStringType" minOccurs="0" maxOccurs="1">
+										   <xsd:annotation>
+												<xsd:documentation>A string the represents the name of a particular group. In Windows, group names are case-insensitive. As a result, it is recommended that the case-insensitive operations are used for this entity. In a domain environment, groups should be identified in the form: "domain\group name". For local groups use: "computer name\group name". For built-in accounts on the system, use the group name without a domain.</xsd:documentation>
+										   </xsd:annotation>
+									  </xsd:element>
+									  <xsd:element name="user" type="oval-def:EntityStateStringType" minOccurs="0" maxOccurs="unbounded">
```

Comment:
State elements should never be "unbounded".  The entity_check attribute governs the evaluation of a single state element against the potentially multiple corresponding item elements.  Multiple states should be used if multiple comparisons are required.

Reply:
Good catch, sorry for yet another round of updates and review.  I'll get better with this eventually